util/gdas_init/set_fixed_files.sh

```diff
@@ -38,7 +38,7 @@ ORO_NAME="${CTAR}.mx${OCNRES}_oro_data"
 #---------------------------------------------------------------------------
 
 if [ "$machine" = 'ursa' ] ; then
-  FIX_ORO_INPUT=/scratch3/NCEPDEV/global/role.glopara/fix/orog/20230615
+  FIX_ORO_INPUT=/scratch3/NCEPDEV/global/role.glopara/fix/orog/20240917
 elif [ "$machine" = 'wcoss2' ] ; then
   FIX_ORO_INPUT=/lfs/h2/emc/global/noscrub/emc.global/FIX/fix/orog/20230615
```

Comment:
Did we want to update the date on WCOSS2 as well?

Reply:
Adding onto this, I think all supported machines should be updated.  I'm not very familiar with jet, but gaea c6 is becoming more widely used for global applications.